src/Endpoints/class-rest-metadata.php

```diff
@@ -101,7 +83,7 @@ public function get_callback( array $object ): array {
 		}
 
 		$response = array(
-			'version' => self::REST_VERSION,
+			'version' => self::VERSION,
```

Comment:
`VERSION` is too generic and needs context. `REST_VERSION` is more descriptive.

Reply:
I agree with what you're saying. However, I intend to use this for versioning GraphQL too and GraphQL is not REST. The alternative that I can see is using a separate version for each, but I think it would introduce too many variables and end up being confusing.